ui/src/testing/stubs/concept-sets-api-stub.ts

```diff
@@ -80,9 +78,7 @@ export class ConceptSetsApiStub extends ConceptSetsApi {
   // TODO when this piece is converted
 
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.